.github/CONTRIBUTING.md

```diff
@@ -1,10 +1,8 @@
-CONTRIBUTING
-============
+# CONTRIBUTING
```

Comment:
The formatting changes here are because I have my editor set up to run prettier on save (could revert if desired). Same in a couple other markdown files.

Reply:
I think we should revert this - danger actually checks for diffs. So this will blow up danger.

Reply:
Not quite sure what you mean about danger?